test-requirements.txt

```diff
@@ -1,7 +1,7 @@
-apache-airflow==1.10.11
+apache-airflow==1.10.12
+apache-airflow[gcp]==1.10.12
 codecov==2.1.8
 marquez-python==0.7.4
 pytest==6.0.1
 pytest-cov==2.10.1
 requests==2.24.0
-psycopg2==2.8.6
```

Comment:
Mind adding this dep back? It's needed for the `PostgresExtractor` test?

Reply:
missed it in merge. fixed, now.

Reply:
thanks 👍